rpi-eeprom-config

```diff
@@ -98,22 +98,26 @@ class BootloaderImage(object):
 
 def main():
     parser = argparse.ArgumentParser(formatter_class=argparse.RawDescriptionHelpFormatter, \
-    description='RPI EEPROM config tool. \
+    description='Bootloader EEPROM configuration tool for the Raspberry Pi 4B. \
 \n\nThere are 3 operating modes: \
 \n\n1. Output the bootloader configuration stored in an EEPROM image file to \
 the screen (STDOUT): specify only the name of an EEPROM image file using the \
-EEPROM option. \
+\'eeprom\' option. \
 \n\n2. Output the bootloader configuration stored in an EEPROM image file to a \
-file: specify the EEPROM image file using the EEPROM option, and the output file\
-using the --out option. \
+file: specify the EEPROM image file using the \'eeprom\' option, and the output \
+file using the \'--out\' option.\
 \n\n3. Insert a new bootloader configuration into an EEPROM image file: specify \
-the EEPROM image file using the EEPROM option, the bootloader configuration file\
-using the --config option, and the name to use for the new file which will be a \
-combination of the original EEPROM image file, along with the new bootloader \
-configuration.')
+the source EEPROM image file using the \'eeprom\' option and the bootloader \
+configuration file using the \'--config\' option. A new file which is a \
+combination of the EEPROM image file, together with the new bootloader \
+configuration file will be created - specify its name using the \'--out\' option. \
+The new bootloader configuration will replace any configuration present in the \
+source EEPROM image.\
+\n\nBootloader EEPROM images are contained in the \'rpi-eeprom-images\' package,\
+ which installs them to the /lib/firmware/raspberrypi/bootloader directory.')
     parser.add_argument('--config', help='Filename of new bootloader config')
     parser.add_argument('--out', help='Name of output file')
-    parser.add_argument('eeprom', help='EEPROM filename (pieeprom.bin)')
+    parser.add_argument('eeprom', help='name of EEPROM file to use as input')
```

Comment:
It looks like all three command-line arguments expect a filename? Maybe it'd be more consistent to use either "name" or "filename" in the help-text for all three?

Reply:
Yes, and I've also removed the word "new" to simplify it, since the explanation of the parameters is not enough on its own to understand how the tool actually works, due to how the parameters operate. The description of the parameters must be read together with the description, which describes the 3 operating modes.

Reply:
Should the 'name of EEPROM file' start with a capital N ?